packages/core/src/physics/joint/HingeJoint.ts

```
@@ -135,35 +131,58 @@
     if (this._limits !== value) {
-      this._limits = value;
-      if (this.useSpring) {
-        (<IHingeJoint>this._nativeJoint).setSoftLimit(value.min, value.max, value.stiffness, value.damping);
-      } else {
-        (<IHingeJoint>this._nativeJoint).setHardLimit(value.min, value.max, value.contactDistance);
+      if (this._limits) {
+        this._limits._updateFlagManager.removeListener(this._onLimitsChanged);
       }
+
+      this._limits = value;
+      value && value._updateFlagManager.addListener(this._onLimitsChanged);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)